### PR TITLE
setup: add a circleci config yml

### PR DESCRIPTION
**Pull request by avinash360**
- add a circleci config yml
- add jest as dev dependency in server

Closes#2

### CODE REVIEW

**Comment by avinash360** [comment]
This pull request depends upon https://github.com/pesto-students/batch-9-Posto/pull/28

File: client/src/serviceWorker.js [hunk 1]
```diff
@@ -0,0 +1,135 @@
+// This optional code is used to register a service worker.
+// register() is not called by default.
+
```
Comment (rishabh1403) [inline client/src/serviceWorker.js]: are you going to use this file ?

**Comment by rishabh1403** [comment]
Please create atomic PRs

**Comment by avinash360** [comment]
@rishabh1403 The file has been removed in another pull request

File: client/src/serviceWorker.js (hunk 1 above)
Comment (avinash360) [reply client/src/serviceWorker.js]: This file has been removed in another pull request, due to commit issues it was shown in my commit.

**Pull request merged**